### DISCUSSION TITLE
Bugfix releases?

### DISCUSSION THREAD

**Discussion by gotson** [jreleaser/jreleaser#1235] [Ideas]
Hi, i know the release schedule is every 2 months, and those releases are packed with new features.

Would it make sense to release intermediary releases during that 2 month gap for bug fixes only?

**Comment by aalmiray** [comment]
I'm wondering about that right now as well, considering the recent set of issues found so far it may be worth pushing `1.5.1` within the next 2 to 3 weeks.

**Reply by aalmiray** [reply]
13 bugs squashed so far in [1.6.0](https://github.com/jreleaser/jreleaser/milestone/24?closed=1). I think renaming the milestone to `v1.5.1` is certainly a possibility now as I'm holding off merging features to the default branch.
